.github/workflows/issue_bot.yml

```
@@ -10,14 +10,40 @@ jobs:
     permissions:
       issues: write
     steps:
+      - name: 'Check Author'
+        uses: actions/github-script@v7
+        id: check-author
+        with:
+          github-token: ${{ secrets.ACCESS_TOKEN }}
+          debug: true
+          script: |
+            const author = context.payload.sender.login;
+            const maintainers = await github.rest.teams.listMembersInOrg({
+                org: owner,
+                team_slug: 'digiwf_maintainer'
+            });
+            const devs = await github.rest.teams.listMembersInOrg({
+                org: owner,
+                team_slug: 'digiwf_dev'
+            });
+            const admins = await github.rest.teams.listMembersInOrg({
+                org: owner,
+                team_slug: 'digiwf_admin'
+            });
+            const coreMembers = maintainers.concat(devs, admins).data.map(u => u.login)
```

```suggestion
            const coreMembers = maintainers.data.concat(devs.data, admins.data).map(u => u.login)
```
Hätte jetzt eher so gesagt